_posts/2016-08-05-lambda.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: Scaling with AWS Lambda
+date: 2016-08-05 20:35:04
+description: Scale your applications with AWS Lambda without having to think about it. This is a case study from a migration done at Lost My Name.
+---
+At [Lost My Name](https://www.lostmy.name/) I'm responsible for the reliability and performance of the platform. This platform is the core of the business as it enables the creatives to publish their work. This work is what makes Lost My Name's products so magical and popular.
+
+As of writing, the team is preparing for another Q4. This will be my second time going through this period at Lost My Name after been through it twice at [Boomf](https://boomf.com/). The experience is unique as you see the sales charts go up and up.
```

Comment:
Misschien ook uitleggen wat er bijzonder is aan "another Q4".
Ik gok dat ge doelt op holiday sales & de daarbijhorende drukte?
Is in elk geval niet meteen/voor iedereen wat er speciaal is aan Q4 :)